tutorials/spanruler_restaurant_reviews/.gitignore

```diff
@@ -0,0 +1,168 @@
+assets/
+corpus/
+training/
+metrics/
+models/
+
+
+# Byte-compiled / optimized / DLL files
```

Comment:
What parts of this `.gitignore` are needed for this particular project subdirectory?

Reply:
Updated!